samples/ControlCatalog/Pages/ItemsRepeaterPage.xaml.cs

```diff
@@ -117,9 +117,8 @@ private void LayoutChanged(object sender, SelectionChangedEventArgs e)
         private void ScrollTo(int index)
         {
             System.Diagnostics.Debug.WriteLine("Scroll to " + index);
-            var layoutManager = ((TopLevel)VisualRoot!).LayoutManager;
             var element = _repeater.GetOrCreateElement(index);
-            layoutManager.ExecuteLayoutPass();
+            ((TopLevel)VisualRoot!).UpdateLayout();
```

Comment:
We should perhaps add `Layoutable.UpdateLayout` like in WPF/UWP:  https://learn.microsoft.com/en-us/dotnet/api/system.windows.uielement.updatelayout?view=windowsdesktop-7.0 which would call `UpdateLayout` on the root.